pkg/agent/cniserver/ipam/antrea_ipam_test.go

```diff
@@ -88,11 +93,62 @@ func createIPPools(crdClient *fakepoolclient.IPPoolClientset) {
 			IPRanges: []crdv1a2.SubnetIPRange{subnetRangeOrange},
 		},
 	})
+
+	ipRangePear := crdv1a2.IPRange{
+		Start: "10.2.3.100",
+		End:   "10.2.3.200",
+	}
+	subnetInfoPear := crdv1a2.SubnetInfo{
+		Gateway:      "10.2.3.1",
+		PrefixLength: 24,
+	}
+	subnetRangePear := crdv1a2.SubnetIPRange{IPRange: ipRangePear,
+		SubnetInfo: subnetInfoPear}
+	crdClient.InitPool(&crdv1a2.IPPool{
+		ObjectMeta: metav1.ObjectMeta{Name: testPear},
+		Spec: crdv1a2.IPPoolSpec{
+			IPRanges:  []crdv1a2.SubnetIPRange{subnetRangePear},
+			IPVersion: 4,
+		},
+		Status: crdv1a2.IPPoolStatus{IPAddresses: []crdv1a2.IPAddressState{{
+			IPAddress: "10.2.3.198",
+			Phase:     crdv1a2.IPAddressPhaseReserved,
+			Owner: crdv1a2.IPAddressOwner{StatefulSet: &crdv1a2.StatefulSetOwner{
+				Name:      "pear-sts",
+				Namespace: testPear,
+				Index:     8,
+			}},
+		}, {
+			IPAddress: "10.2.3.197",
+			Phase:     crdv1a2.IPAddressPhaseReserved,
+			Owner: crdv1a2.IPAddressOwner{
+				Pod: &crdv1a2.PodOwner{
+					Name:        "pear-sts-9",
+					Namespace:   testPear,
+					ContainerID: "pear-sts-9-container",
+				},
+				StatefulSet: &crdv1a2.StatefulSetOwner{
+					Name:      "pear-sts",
+					Namespace: testPear,
+					Index:     9,
+				}},
+		}, {
+			IPAddress: "10.2.3.196",
+			Phase:     crdv1a2.IPAddressPhaseReserved,
+			Owner: crdv1a2.IPAddressOwner{
+				Pod: &crdv1a2.PodOwner{
+					Name:        "pear10",
+					Namespace:   testPear,
+					ContainerID: "pear10-container",
+				}},
+		}}},
+	})
 }
 
 func initTestClients() (*fake.Clientset, *fakepoolclient.IPPoolClientset) {
 	crdClient := fakepoolclient.NewIPPoolClient()
 
+	true := true
```

Comment:
`bTrue := true`, better not to override the keywords, which could make the following code that use the variables confusing

Reply:
Fixed.